### PR TITLE
docs: fix prerender

### LINKED ISSUE #1987: [DOC] Broken URL for "@nuxt/content theme docs"
### Environment

The URL redirecting to `@nuxt/content theme docs` i.e. `https://content.nuxtjs.org/themes-docs` is broken & returns 404. Even tried with an alternate URL (`https://content.nuxtjs.org/en/v1/examples/docs-theme`), it also returns the same.

### Reproduction

Visit [`https://content.nuxtjs.org/themes-docs`](https://content.nuxtjs.org/themes-docs) or [`https://content.nuxtjs.org/en/v1/examples/docs-theme`](https://content.nuxtjs.org/en/v1/examples/docs-theme) to inspect.

### Describe the bug

URL is broken & returns 404 for [`en`](https://content.nuxtjs.org/en/v1/examples/docs-theme) or [main](https://content.nuxtjs.org/themes-docs), while the [`ja`](https://content.nuxtjs.org/ja/v1/examples/docs-theme) returns the page.

### Additional context

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by farnabaz**
### 🔗 Linked issue
resolves #1987
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by nobkd** [comment]
You accidentally deleted almost all starting parentheses **`(`** of the links

**Comment by farnabaz** [comment]
@nobkd Hah nice catch 🙂 
Thanks for mentioning that, I didn't double-check the changes on local 😀

File: docs/content/3.guide/5.migration/1.from-v1.md [hunk 1]
```diff
@@ -102,8 +102,8 @@ The `<ContentDoc>` component will fetch the document for the current route path
...
 | Reactive Composables                                |                             ❌                             |                         ✅                        |
 | [FrontMatter](/guide/writing/markdown#front-matter) |                             ✅                             |                         ✅                        |
-| [Excerpt](/writing#excerpt)                         |                             ✅                             |                         ✅                        |
-| [Table Of Contents](/writing#table-of-contents)     |                             ✅                             |                         ✅                        |
+| [Excerpt]/v1/getting-started/writing#excerpt)                         |                             ✅                             |                         ✅                        |
+| [Table Of Contents]/v1/getting-started/writing#table-of-contents)     |                             ✅                             |                         ✅                        |
```
Comment (nobkd) [inline docs/content/3.guide/5.migration/1.from-v1.md:106]:
```suggestion
| [Excerpt](/v1/getting-started/writing#excerpt)                         |                             ✅                             |                         ✅                        |
| [Table Of Contents](/v1/getting-started/writing#table-of-contents)     |                             ✅                             |                         ✅                        |
```

**Review by nobkd — CHANGES_REQUESTED** [review]
Probably missed this one?

**Pull request merged**